README.md

```diff
@@ -2,21 +2,33 @@
 
 
 ### Description
-A Python Application for a slackbot that routes text requests and gets responses from Watson Assistant
+A Python Application for a slackbot that routes text requests and gets responses from Watson Assistant Solutions.
 
-[![License](https://img.shields.io/badge/license-APACHE2-blue.svg)]() [![Python](https://img.shields.io/badge/Python-3.6.2-yellow.svg)]()
+[![Build Status](https://travis.ibm.com/ConsumerIoT/simple_WPA_slackbot.svg?token=zZTqm68ChrVG6ms39mnF&branch=master)](https://travis.ibm.com/ConsumerIoT/simple_WPA_slackbot) [![License](https://img.shields.io/badge/license-APACHE2-blue.svg)]() [![Python](https://img.shields.io/badge/Python-3.6.2-yellow.svg)]() [![Version](https://img.shields.io/badge/Version-3.1.3-green.svg)]()
 
 ---
 
 ### Requirements:
 
-* sys
-* requests
-* python-dotenv
-* slackclient
+* websocket-client==0.47.0
+* python-dotenv>=0.9.1
+* requests>=2.19.1
+* slackclient>=1.2.1
 
 ---
 
+### Getting Your Slack Key
```

Comment:
I set up a meeting for tomorrow to go through this part as I don't see these choices in my slack workgroup.

Reply:
had meeting and determined steps to be 

1. Go to https://YOUR_SLACK.slack.com/apps/manage
2. In the __Search App Directory__ field at the top, type __bots__
3. Click on __Bots__
4. Click on the __Add Configuration__ button
5. Give your bot a unique __username__ (Save this info)
6. Save the __API Token__, 
7. Create your __.env__ file as instructed in the below steps